program/rust/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+cargo-features = ["edition2021"]
+
+[package]
+name = "pyth-oracle"
+version = "0.1.0"
```

Comment:
I think we're on 2.13.0 right now https://github.com/pyth-network/pyth-client/releases , so probably bump that to 2.13.1